### PR TITLE
WIP: Allow translations in module.

### PR DESCRIPTION
**Pull request by krystian8207**
# Allow translations in module

On #48. Fixed via adding `ns` in each module.

### CODE REVIEW

File: R/ui.R [hunk 1]
```diff
@@ -50,7 +51,10 @@ i18n_state <- function(init_language) {
 #' @importFrom jsonlite toJSON
 #' @import glue
 #' @export
-usei18n <- function(translator) {
+usei18n <- function(translator, ns = NULL) {
+  if (is.null(ns)) {
+    ns <- function(id) id
```
Comment (dokato) [inline R/ui.R:56]: `ns <- identity` ?

Comment (dokato) [reply R/ui.R:56]: and let's move it to `i18n_state` where it's actually used

**Comment by dokato** [comment]
@krystian8207 what about solution inspired by shinyjs here:
```r
id <- ifelse(inherits(session, "session_proxy"), session$ns(id), id)
```

**Comment by przytu1** [comment]
Solved by https://github.com/Appsilon/shiny.i18n/pull/62

**Pull request closed without merging**